samples/system-test/kms.test.js

```diff
@@ -301,7 +301,7 @@ describe('kms sample tests', () => {
   });
 
   describe('asymmetric keys', () => {
-    const kms = require('../../src');
+    const kms = require('@google-cloud/kms');
```

Comment:
I don't think that's a thing we want to change.  We don't run `npm link` before running system tests, so I'm not quite sure what this would do to be honest.

Reply:
@JustinBeckwith Those are samples tests, we do run `npm link` there, don't we?

Reply:
Ooooops.  I thought this was just a system test, and misread the path.  This is fine.  Apologies!